### PR TITLE
Improvements and bugfixes settlement run

### PR DESCRIPTION
**Pull request by xee8ai**
Has to be checked with data at other installations…

### CODE REVIEW

File: modules/BillingBase/Console/SettlementRunCommand.php [hunk 1]
```diff
@@ -618,6 +618,12 @@ protected function _parse_envia_csv($filepath)
                 'price' 	=> str_replace(',', '.', $arr[10]),
                 ];
 
+            // extend $data for other providers
+            // attention: envia sends latin-1 – we have to convert to UTF-8…
+            if ($arr[11] != '016-envia TEL') {
+                $data['called_nr'] = iconv('CP1252', 'UTF-8', '\\emph{anderer Anbieter:}\\newline\textbf{'.$arr[11].'}\\newline~~'.$arr[7].'\\newline~~'.$arr[8].'\\newline~~'.$arr[9]);
```
Comment (NinoRy) [inline modules/BillingBase/Console/SettlementRunCommand.php]: Is it possible to use an array here and build the latex syntax in the latex template? I dont like to put latex syntax directly into the code as this should be independent of the template syntax. We could somewhen consider to build the template as libre/open/microsoft office document as well.

Comment (xee8ai) [reply modules/BillingBase/Console/SettlementRunCommand.php]: Accepted and changed.

**Comment by xee8ai** [comment]
Should we change
list($a, $b) = [1, 2]
to
[$a, $b] = [1, 2]
(possible >=PHP7.1) or change our StyleCI settings?

**Comment by NinoRy** [comment]
pls fix style issues!

**Comment by xee8ai** [comment]
Rebased to dev to solve StyleCI issues.

@cschra: Could you give me a hint what needs to be adapted?

**Comment by cschra** [comment]
Was just a visual reminder that there are still things open ... You can remove the label and add "Ready to review" if you like

**Pull request merged**